simularium/simularium-viewer
# move webpack to example viewer

Time Estimate or Size
=======
small 

Problem
=======
The test bed viewer app is the only thing that needs web pack but we still had it in our main package.json 
closes #358 

Solution
========
removed all web packfrom main code 

## Type of change
Please delete options that are not relevant.

* Maintenance 


Steps to Verify:
----------------
1. do a fresh install of node modules
2. `npm run build` should work 
3. `npm start` should open up test bed viewer
